src/functions-and-arrays.js

```diff
@@ -52,16 +118,45 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(arr) {
+  let noDuplicates = [];
+  if (arr.length === 0) {
+    return null;
+  } else if (arr === undefined) {
+    return null;
+  }
+  for (let i = 0; i < arr.length; i++) {
+    if (noDuplicates.indexOf(arr[i]) === -1) {
+      noDuplicates.push(arr[i]);
+    } else {
+      continue;
+    }
+  }
+  return noDuplicates;
+}
 
+const noDuplicates = uniquifyArray(wordsUnique);
+console.log(noDuplicates);
 
 // Iteration #6: Find elements
```

Comment:
Iteration 6 task says, ` Don't use indexOf for this one.`. Can you think of an alternative way to solve this ? ;)

Reply:
function doesWordExist(arrOfWords, word) {
  if (arrOfWords.length === 0) {
    return null;
  }
  for (let i = 0; i < arrOfWords.length; i++) {
    if (arrOfWords[i].includes(word) === true) {
      return true;
    }
  }
  return false;
}

Reply:
I read it wrong oops!